packages/y/yuzu/package.yml

```diff
@@ -12,8 +12,8 @@ description: |
 builddeps  :
     - pkgconfig(Qt5Multimedia)
     - pkgconfig(fmt)
+    - pkgconfig(gamemode)
```

Comment:
Why? Is this an actual added dependency from upstream?

Reply:
My understanding is that this enables automatic gamemode integration for users with gamemode installed. See https://github.com/yuzu-emu/yuzu/issues/10078